multiapps-common/src/main/java/org/cloudfoundry/multiapps/common/MultiappsImmutablesStyle.java

```diff
@@ -4,9 +4,8 @@
 
 /**
  * Used for generation of code via <a href="https://immutables.github.io/">Immutables</a>.
- *
  */
-@Value.Style(depluralize = true, jdkOnly = true, allowedClasspathAnnotations = java.lang.Override.class)
+@Value.Style(depluralize = true, jdkOnly = true, allowedClasspathAnnotations = java.lang.Override.class, visibility = Value.Style.ImplementationVisibility.PUBLIC, typeImmutableNested = "Immutable*")
```

Comment:
the one thing that's missing from the previous style config is
 > typeImmutable = "Immutable*"
 in addition to typeImmutableNested. Do we not need it?

Reply:
Default value is matching previously set configuration.